tests/unit_tests/system/test_calculate_RR0.cpp

```diff
@@ -39,34 +34,31 @@ TEST(CalculateRR0Tests, OneNode) {
 }
 
 TEST(CalculateRR0Tests, TwoNodes) {
-    const auto r0 = Kokkos::View<double[1][2][4]>("r0");
-    constexpr auto r0_host_data = std::array{1., 2., 3., 4., 5., 6., 7., 8.};
-    const auto r0_host = Kokkos::View<const double[1][2][4], Kokkos::HostSpace>(r0_host_data.data());
-    const auto r0_mirror = Kokkos::create_mirror(r0);
-    Kokkos::deep_copy(r0_mirror, r0_host);
-    Kokkos::deep_copy(r0, r0_mirror);
-
-    const auto r = Kokkos::View<double[1][2][4]>("r");
-    constexpr auto r_host_data = std::array{5., 6., 7., 8., 9., 10., 11., 12.};
-    const auto r_host = Kokkos::View<const double[1][2][4], Kokkos::HostSpace>(r_host_data.data());
-    const auto r_mirror = Kokkos::create_mirror(r);
-    Kokkos::deep_copy(r_mirror, r_host);
-    Kokkos::deep_copy(r, r_mirror);
+    const auto x = Kokkos::View<double[1][2][7]>("x");
+    constexpr auto x_host_data =
+        std::array{1., 2., 3., 4., 5., 6., 7., 8., 9., 10., 11., 12., 13., 14.};
+    const auto x_host = Kokkos::View<const double[1][2][7], Kokkos::HostSpace>(x_host_data.data());
+    const auto x_mirror = Kokkos::create_mirror(x);
+    Kokkos::deep_copy(x_mirror, x_host);
+    Kokkos::deep_copy(x, x_mirror);
 
     const auto rr0 = Kokkos::View<double[1][2][6][6]>("rr0");
 
-    Kokkos::parallel_for("CalculateRR0", 2, CalculateRR0{0, r0, r, rr0});
-
-    constexpr auto expected_rr0_data =
-        std::array{2780.,   4400.,  -400.,   0.,     0.,      0.,      -3280.,  2372.,   3296.,
-                   0.,      0.,     0.,      2960.,  -1504.,  4028.,   0.,      0.,      0.,
-                   0.,      0.,     0.,      2780.,  4400.,   -400.,   0.,      0.,      0.,
-                   -3280.,  2372.,  3296.,   0.,     0.,      0.,      2960.,   -1504.,  4028.,
-
-                   16412.,  74896., -11984., 0.,     0.,      0.,      -27376., 17284.,  70528.,
-                   0.,      0.,     0.,      70736., -10688., 30076.,  0.,      0.,      0.,
-                   0.,      0.,     0.,      16412., 74896.,  -11984., 0.,      0.,      0.,
-                   -27376., 17284., 70528.,  0.,     0.,      0.,      70736.,  -10688., 30076.};
+    Kokkos::parallel_for("CalculateRR0", 2, CalculateRR0{0, x, rr0});
+
+    constexpr auto expected_rr0_data = std::array{-44.,  4.,   118., 0.,    0.,   0.,    //
```

Comment:
Does the `//` prevent the clang-format from ruining the matrix layouts?  If so, that is magical.

Reply:
Yes, the `//` forces clang-format to keep the line break where it should be, which makes the matrix so much more readable.